src/assets/scss/components/_search.scss

```diff
@@ -6,18 +6,14 @@
   }
 
   &__container {
-    padding-bottom: 1rem;
-    margin-bottom: 1rem;
-    max-width: 40rem;
-
     button {
       font-size: 1rem;
       margin: .25rem .5rem .25rem 0;
```

Comment:
```suggestion
      margin: .25rem .5rem .25rem 0;
      border: 1px solid black;
```

Reply:
Attached the screenshot above.

Reply:
Just changed this to pink on feedback from the client :)